pkg/operation/botanist/matchers/matcher.go

```diff
@@ -46,124 +46,135 @@ import (
 )
 
 var (
-	kubeSystemLabels = labels.Set{
+	defaultNamespaceLabels = labels.Set{
+		corev1.LabelMetadataName: metav1.NamespaceDefault,
+	}
+
+	kubeSystemNamespaceLabels = labels.Set{
 		v1beta1constants.ShootNoCleanup:  "true",
 		v1beta1constants.GardenerPurpose: metav1.NamespaceSystem,
+		corev1.LabelMetadataName:         metav1.NamespaceSystem,
```

Comment:
Doesn't this mean that webhooks now need all three labels so that they are not considered problematic? 👀

Reply:
One of these labels for exclusion is already enough for the [match](https://github.com/gardener/gardener/blob/3840acaaf57955fd65330c83b2b2d5bdaad56179/pkg/operation/botanist/matchers/matcher.go#L224). This makes it easier to write a single rule which excludes `kube-system` and `default`, i.e.

```
namespaceSelector:
  matchExpressions:
  - key: kubernetes.io/metadata.name
    operator: NotIn
    values:
    - kube-system
    - default
```

Reply:
Okay, perfect, wasn't sure about this one. Thanks!

Reply:
Thanks @ialidzhikov, will do.